src/main/java/log_in/use_case/LogInInteractor.java

```diff
@@ -22,10 +22,10 @@ public class LogInInteractor implements LogInInputBoundary {
      * @param access to retrieve stuff from the database
      * @param outputBoundary contains the result of this use case
      */
-    public LogInInteractor(LogInDsGateway access, LogInOutputBoundary outputBoundary, UserFactory userFactory){
+    public LogInInteractor(LogInDsGateway access, LogInOutputBoundary outputBoundary, UserFactory factory){
```

Comment:
What is the point of having `factory` as a parameter if you're not going to use it?

Reply:
ah nice catch, I forgot to assign it 
thanks